mage_ai/data_cleaner/statistics/calculator.py

```diff
@@ -231,9 +235,11 @@ def __statistics_overview(self, series, col):
             )
 
         # Detect mismatched formats for some column types
-        invalid_rows = get_mismatched_rows(series_non_null, column_type)
-        data[f'{col}/invalid_value_count'] = len(invalid_rows)
-        data[f'{col}/invalid_values'] = invalid_rows[:INVALID_VALUE_SAMPLE_COUNT]
+        invalid_rows = find_syntax_errors(series_non_null, column_type)
+        data[f'{col}/invalid_value_count'] = invalid_rows.sum()
+        invalid_values = series_non_null[invalid_rows]
+        data[f'{col}/invalid_values'] = invalid_values[:INVALID_VALUE_SAMPLE_COUNT].tolist()
+        data[f'{col}/invalid_indices'] = invalid_values.index.tolist()
```

Comment:
For invalid indices, we need to return all indices below [SAMPLE_SIZE](https://github.com/mage-ai/mage-ai/blob/master/mage_ai/server/data/models.py#L7).

Reply:
Ok, is there any way to support sharing that constant between the `server` subpackage and the `data_cleaner` subpackage? If not, I could add a new `shared` subpackage in the `mage_ai` folder from which both `server` and `data_cleaner` could fetch this constant value.

Reply:
just move the constant to data_cleaner subpackage

Reply:
Okay I moved this constant into the shared folder in the data_cleaner subpackage.